src/OpenTelemetry.Api/Internal/HttpSemanticConventionHelper.cs

```diff
@@ -0,0 +1,67 @@
+// <copyright file="HttpSemanticConventionHelper.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Internal
+{
```

Comment:
just one more nit. use a file-scoped namespace here to reduce indentation

Reply:
no problem!

any way to make this a code style rule for future PRs?

Reply:
We should also update the existing classes to use file-scoped namespace for being consistent.